File: GCFinalProject/Models/Character.cs
using System;
using System.Collections.Generic;

namespace GCFinalProject.Models
{
    public partial class Character
    {
        public Character()
        {
            Favorites = new HashSet<Favorite>();
        }

        public int PkId { get; set; }
        public string? Class { get; set; }
        public string? Subclass { get; set; }
        public int? Strength { get; set; }
        public int? Dexterity { get; set; }
        public int? Constitution { get; set; }
        public int? Intelligence { get; set; }
        public int? Wisdom { get; set; }
        public int? Charisma { get; set; }

        public virtual ICollection<Favorite> Favorites { get; set; }
    }
}

File: GCFinalProject/ClientApp/src/app/character.ts
export class Character{
pkId: number;
class: string;
subclass: string;
strength: number;
dexterity: number;
constitution: number;
intelligence: number;
wisdom: number;
charisma: number;


constructor(pkId: number, Class: string, subclass: string, strength: number, dexterity: number, constitution: number, intelligence: number,wisdom: number, charisma: number){
this.pkId = pkId;
this.class = Class;
this.subclass = subclass;
this.strength = strength;
this.dexterity = dexterity;
this.constitution = constitution;
this.intelligence = intelligence;
this.wisdom = wisdom;
this.charisma = charisma;

}



}